### PR TITLE
[net] Distinguish some identical debug messages; print rvals

### PR DESCRIPTION
**Pull request by grgustaf**
Signed-off-by: Geoff Gustafson <geoff@linux.intel.com>

### CODE REVIEW

File: src/zjs_net.c [hunk 1]
```diff
@@ -1253,10 +1253,12 @@ static ZJS_DECL_FUNC(socket_connect)
...
-                sizeof(peer_addr6), tcp_connected, 1, handle) < 0) {
-            DBG_PRINT("connect failed\n");
+        int rval = net_context_connect(handle->tcp_sock,
+                                       (struct sockaddr *)&peer_addr6,
+                                       sizeof(peer_addr6), tcp_connected,
+                                       1, handle);
+        if (rval < 0) {
+            DBG_PRINT("IPv6 connect failed: %d\n", rval);
             zjs_obj_add_boolean(this, false, "connecting");
             error_desc_t desc = create_error_desc(ERROR_CONNECT_SOCKET, this,
```
Comment (jprestwo) [inline src/zjs_net.c]: you didn't edit this line but what is this doing? Its just returning so `desc` isn't being used. Shouldn't the `error` event get emitted like it does below?

Comment (grgustaf) [reply src/zjs_net.c]: Did you get confused because the patch cuts off here? It does call defer_emit in the full source.

Comment (jprestwo) [reply src/zjs_net.c]: yes, I only pressed the expand button once. Disregard.

**Comment by brianjjones** [comment]
LGTM +1

**Pull request merged**